src/history/hs.c

```diff
@@ -1133,7 +1189,7 @@ __wt_hs_find_upd(WT_SESSION_IMPL *session, WT_ITEM *key, const char *value_forma
     WT_ITEM hs_key, recno_key;
     WT_MODIFY_VECTOR modifies;
     WT_TXN *txn;
-    WT_TXN_SHARED *txn_shared;
+    WT_TXN_SHARED *txn_shared_local;
```

Comment:
Why is the variable name change here?

Reply:
It was to avoid shadowing a variable in the `WT_WITH_TXN_ISOLATION` macro.
Since everything is wrapped in functions now, we can probably get rid of this.

Reply:
5956cea